ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -46,6 +46,9 @@ final class CommandsValidator(ledgerId: LedgerId) {
       appId <- requireLedgerString(commands.applicationId, "application_id")
         .map(domain.ApplicationId(_))
       commandId <- requireLedgerString(commands.commandId, "command_id").map(domain.CommandId(_))
+      // TODO: Let the user provide this information in the `ProtoCommands`,
+      //       and extract it from there.
```

Comment:
Is that already planned? If so, I think we can remove this TODO. If not, then I'd rather have a more general comment or none at all.

Reply:
Yes, it is. I'll remove it.